service/src/main/resources/application-feature-toggle.yml

```diff
@@ -0,0 +1,4 @@
+spring:
```

Comment:
Should not be committed, the extension is wrong and as such has not been picked up by the gitignore `yml` -> `yaml`

Reply:
oops 👍